Tutorial - Simplifying

babakfp opened the pull request:
Hi
It doesn't need to run tests or stuff like that. It's just a simple tweak here and there.

Before:
![Screenshot 2021-04-24 015814](https://user-images.githubusercontent.com/44144724/115954747-899a0c00-a4a7-11eb-9ffb-2111291b040b.png)
After:
![Screenshot 2021-04-24 015846](https://user-images.githubusercontent.com/44144724/115954752-8dc62980-a4a7-11eb-99ab-2e7750fb9c4f.png)

dummdidumm commented:
I appreciate your desire to make the tutorial looking a bit better. You talk about simplification but switch the colors from named to hex, which is the opposite. The other adjustments don't really change anything. I'm therefore going to close this.

babakfp commented:
It seems that you a little rushed on this one.
- HEX colors are just some nonsense and no one understands or tries to read them.
- Noone uses colors like `gold, indigo, salmon`.
What do you see in these changes? is it just changing colors to HEX or do they really make a difference to the end-user?

I reduced 1 item from the `things` array because there was more than enough.
I changed the color, not just HEX, I changed the code to make a difference in the UI for the end-user. So colors are simple `Blue, Green, Orange, Red`. The most known colors in the universe. Tell me that you can look at current colors and name them one by one straight away. They were just confusing!. It was like I do need to remember the colors and...
I changed the HTML tags because the rendered content wasn't a paragraph.
I fixed a typo.
A little format.
Later I replaced red and orange with each other.

@dummdidumm

antony commented:
Hi @babakfp - it's a bit annoying to read that you believe somebody "rushed" to close something when actually it was considered in an internal discussion which involved 5 or so people, where we concluded that the changes, whilst well intentioned, were not what we wanted to do.

So thank you for your PR but please accept that these aren't the right changes for now.

babakfp commented:
> Hi @babakfp - it's a bit annoying to read that you believe somebody "rushed" to close something when actually it was considered in an internal discussion which involved 5 or so people, where we concluded that the changes, whilst well intentioned, were not what we wanted to do.
> 
> So thank you for your PR but please accept that these aren't the right changes for now.

Hi
Sure. I was just trying to help.
I did those changes because the colors were a little distracting for me, also it was hard to keep an eye on the boxes to see which one of them going to get removed.
I called it rushed because Mr. Simon said: @dummdidumm
> You talk about simplification but switch the colors from named to hex, which is the opposite.
This was annoying to me because it didn't sound reasonable, it was more like an excuse.
Have a good day/night
